### PR TITLE
Extended usage errors

### PR DESCRIPTION
**Pull request by ewoutp**
All CLI fatal errors (that can be fixed with advise) have a nice looking error message with instructions on how to fix the problem.

### CODE REVIEW

File: help.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+	)
+}
+
+// Arangod is not found at given path.
+func showArangodExecutableNotFoundHelp(arangodPath string) {
+	showFatalHelp(
+		fmt.Sprintf("Cannot find `arangod` (expected at `%s`).", arangodPath),
+		"",
+		"How to solve this:",
+		"1 - Install ArangoDB locally or run the ArangoDB starter in docker. (see RADME for details).",
```
Comment (neunhoef) [inline help.go]: Typo: RADME. If we wanted, we could add that it is possible to run the starter from a source tree in which ArangoDB has been compiled in a subdirectory `build`. However, this is for developers only. One could also add that one can point the starter to the executable and javascript library path with `--server.arangod` and `--server.js-dir` options.

File: help.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	)
+}
+
+// ArangoSync is not found at given path.
+func showArangoSyncExecutableNotFoundHelp(arangosyncPath string) {
+	showFatalHelp(
+		fmt.Sprintf("Cannot find `arangosync` (expected at `%s`).", arangosyncPath),
+		"",
+		"How to solve this:",
+		"1 - Install ArangoSync locally or run the ArangoDB starter in docker. (see RADME for details).",
```
Comment (neunhoef) [inline help.go]: Typo: RADME.

Comment (ewoutp) [reply help.go]: Done

File: help.go (hunk 1 above)
Comment (ewoutp) [reply help.go]: Typo: Done
IMO the build suggestion is a step to far. This PR is intended to simplify and I fear that would do the opposite.

**Pull request merged**